src/stylesheets/article.css

```
@@ -6,3 +6,10 @@
 .article-preview:hover h3 {
   text-decoration: underline;
 }
+.article-preview:hover .img {
```

```suggestion
.item-episode:hover .img,
.article-preview:hover .img {
  transform: scale(1.05);
}
```

But transition is missing 

```
  transition: ease-in-out 300ms;
```

have a look at https://codepen.io/dwinnbrown/pen/kXEEAa